prime-router/src/main/kotlin/cli/tests/BasicTests.kt

```diff
@@ -194,8 +194,9 @@ class End2EndUniversalPipeline : CoolTest() {
 
             // check that lineages were generated properly
             passed = passed and pollForLineageResults(
-                reportId, allGoodReceivers,
-                1, // for UP there should be one item per step
+                reportId,
+                expectedReceivers,
```

Comment:
I did not dig into it too much, but something about how the tests running in parallel configured state meant that this would pass even though it should have failed.